frame/atomic-swap/src/tests.rs

```diff
@@ -30,7 +30,10 @@ frame_support::construct_runtime!(
 
 parameter_types! {
 	pub BlockWeights: frame_system::limits::BlockWeights =
-		frame_system::limits::BlockWeights::simple_max(1024);
+		frame_system::limits::BlockWeights::simple_max(frame_support::weights::WeightV2 {
+			computation: 1024,
+			bandwidth: 1024,
```

Comment:
really not a fan of `bandwidth`. Out of the box, it is much more commonly used to refer to network bandwidth than storage. I would call it `storage` or `IO` (in the spirit of `sp_io`).

Reply:
`proof` or `proof_len` seems the most suitable to me. 

The fact that I had to postfix it with `len` also is pointing out at another minor issue here, that `computation` and `bandwidth` are of different nature, one being `time` and one `byte size`. Just thinking out loud, perhaps we could have something like:

```
struct WeightV2 {
  // things that contribute to the time constraint.
  time: { computation: u64, ... }
  // things that contribute to the bandwidth bandwidth.
  badnwidth: { proof: u64 }
}
```